content/blog-posts/2023-04-01-excelbpf/index.md

```diff
@@ -0,0 +1,28 @@
+---
+path: '/blog/launching-excel-bpf'
+date: '2023-04-01T10:00:00.000Z'
+title: 'Annoucing eBPF 2.0: excelBPF'
+description: 'Hear about the latest innovation from the eBPF community, turn your cells into hives of activity'
+ogImage: 
+toc: false
+categories:
+  - Community
+---
+
+Author: Bill Mulligan - Isovalent
+
+Today the eBPF community is excited to announce the launch of the next version of [eBPF](https://ebpf.io/), excelBPF. Excel is the most widely used programming language in the world and underpins the functioning of many of the most critical institutions in the world. Bringing power of BPF to spreadsheets will enable millions of people to leverage this production proven technology, making bytecode only one cell away.
+
+eBPF has been deployed at scale for over half a decade now, delivering you every cat picture on Facebook, processing and securing your Google queries, and enabling you to binge shows on Netflix without waiting for it to load. These initial adopters showed us the impact eBPF can have on performance and innovation in areas like networking, observability, security, tracing, and profiling. excelBPF will now bring these benefits to many new use cases like finance, inventory management, consulting, and planning your next vacation.
+
+“When we got the first version of eBPF merged into the Linux kernel in 2014, we had ideas about how transformative the technology could be for the Linux kernel, but they were still to be proven out. Seeing the massive step change in innovation and flexibility eBPF has enabled for the kernel community, I’m excited to see what excelBPF holds in store for the world. I think we are really just scratching the surface of what is possible. It feels like Day 1 all over again.” said Daniel Borkmann, co-creator of eBPF.
+
+Looking at where eBPF 1.0 has transformed the world shows us some of the potential excelBPF has for the future. In the cloud native world, [eBPF has unlocked innovation](https://www.infoworld.com/article/3689690/how-ebpf-unlocks-cloud-native-innovation.html) by shortening production feedback loops, extended functionality with a safety belt, provided new building blocks while still leveraging existing tooling, allowing for on the fly upgrades, making data processing faster with lower resource consumption, and creating better abstractions for observability and security. excelBPF is bringing these benefits outside the IT department. Spreadsheets no longer need to be held back by decades old macros.
```

Comment:
The first one is a long sentence, with a mix of (at the beginning) present perfect and (near the end) -ing forms. Maybe split in two?